aws-gen.opam

```
@@ -22,6 +22,8 @@ depends: [
   "yojson" {>= "1.6.0"}
   "ocaml-compiler-libs" {>= "0.11.0" }
   "ocamlgraph"
+  "graphlib"
+  "core"
```

```suggestion
  "graphlib" {>= "2.5.0"}
  "core_kernel" {>= "v0.14" & < "v0.16"}
```
 Looks like we can avoid the full core dependency here plus some version bounds.